web/pages/api/profile/index.js

```diff
@@ -0,0 +1,18 @@
+import nextConnect from 'next-connect';
+import { query } from '../../../utils/query';
+
+const handler = nextConnect().get(async (req, res) => {
+	const user = req.user;
+
+	try {
+		const id = user.id;
+		var [results] = await query(`SELECT * FROM PROFILE WHERE userId = ?`, [id]);
+		if (results.length < 1) res.status(401).json({ error: id });
```

Comment:
I'd prefer if we check `results.length > 0` or `results.length == 0` instead of < 1 to keep things consistent when possible.